CRM/Contact/Import/Form/MapField.php

```diff
@@ -782,19 +758,19 @@ public function submit($params, $mapperKeys) {
       if (($first == 'a' && $second == 'b') ||
         ($first == 'b' && $second == 'a')
       ) {
-        $relatedVal = $this->_mapperFields[$fldName];
+        $parserParameters['mapperRelated'][$i] = $this->_mapperFields[$fldName];
```

Comment:
I changed this to match the other function's name - not sure it makes any more sense though :-)